packages/client/webapp/src/views/TA2Playground.vue

```diff
@@ -0,0 +1,343 @@
+				source = null;
+				target = null;
+			}
+
+			if (source && target) {
+				this.addEdge(source, target);
+				source = null;
+				target = null;
+			}
+		});
+
+		// Test
+		const test = await fetch('http://localhost:8888/api/models', { method: 'PUT' });
+		const testData = await test.json();
+		modelId = testData.id;
+
+		this.refresh();
+		this.jsonOutput();
+	},
+	methods: {
+		async refresh() {
+			await renderer.setData(g);
+			await renderer.render();
+		},
+		async jsonOutput() {
+			const resp = await fetch(`http://localhost:8888/api/models/${modelId}/json`, {
+				method: 'GET'
+			});
+			const output = await resp.json();
+			d3.select('#output').text(JSON.stringify(output, null, 2));
+		},
+		// eslint-disable-next-line
+		async addEdge(source: any, target: any) {
+			g.edges.push({
+				source: source.datum().id,
+				target: target.datum().id,
+				points: [
+					{
+						x: source.datum().x + source.datum().width * 0.5,
+						y: source.datum().y + source.datum().height * 0.5
+					},
+					{
+						x: target.datum().x + target.datum().width * 0.5,
+						y: target.datum().y + target.datum().height * 0.5
+					}
+				]
+			});
+
+			await fetch(`http://localhost:8888/api/models/${modelId}`, {
+				method: 'POST',
+				headers: {
+					Accept: 'application/json',
+					'Content-Type': 'application/json'
+				},
+				body: JSON.stringify({
+					edges: [
+						{
+							source: source.datum().id,
+							target: target.datum().id
+						}
+					]
+				})
+			});
+
+			// g = runLayout(_.cloneDeep(g));
+			this.refresh();
+			this.jsonOutput();
+		},
+		async addPlace() {
+			console.log('add place');
+			placeCounter++;
+			const id = `p-${placeCounter}`;
+
+			g.nodes.push({
+				id,
+				label: id,
+				x: Math.random() * 400,
+				y: Math.random() * 400,
+				height: 50,
+				width: 50,
+				data: { type: 'species' },
+				nodes: []
+			});
+			this.refresh();
+
+			await fetch(`http://localhost:8888/api/models/${modelId}`, {
+				method: 'POST',
+				headers: {
+					Accept: 'application/json',
+					'Content-Type': 'application/json'
+				},
+				body: JSON.stringify({
+					nodes: [
+						{
+							name: id,
+							type: 'S'
+						}
+					]
+				})
+			});
+			this.jsonOutput();
+		},
+		async addTransition() {
+			console.log('add transition');
+			transitionCounter++;
+			const id = `t-${transitionCounter}`;
+
+			g.nodes.push({
+				id,
+				label: id,
+				x: Math.random() * 400,
+				y: Math.random() * 400,
+				height: 50,
+				width: 50,
+				data: { type: 'transition' },
+				nodes: []
+			});
+			this.refresh();
+
+			await fetch(`http://localhost:8888/api/models/${modelId}`, {
+				method: 'POST',
+				headers: {
+					Accept: 'application/json',
+					'Content-Type': 'application/json'
+				},
+				body: JSON.stringify({
+					nodes: [
+						{
+							name: id,
+							type: 'T'
+						}
+					]
+				})
+			});
+			this.jsonOutput();
+		}
+	}
+});
+</script>
+
+<template>
+	<div style="margin: 10px">
```

Comment:
I'll prefer a `<section>` here.
And also to put all the CSS within a `<script scoped>`